modules/special-rules/pages/regroup.adoc

```diff
@@ -0,0 +1,9 @@
+= Regroup
+
+Space Marines are known for their stubborn determination, and yet troops like the Dark Angels Deathwing have a special reputation for regrouping effectively, no matter the circumstances.
+
+---
+
+Apply these rules to detachments with the {regroup} ability:
```

Comment:
Just spotted this older phrasing.

```suggestion
Where your detachment has the {regroup} ability:
```

Reply:
Yeah, I've fixed a couple of those in other commits too. I'll go ahead and apply this one, and get a look at the rest of the text another time.